api/src/main/java/io/strimzi/api/kafka/model/kafka/entityoperator/EntityTopicOperatorSpec.java

```diff
@@ -95,26 +97,32 @@ public int getReconciliationIntervalSeconds() {
     public void setReconciliationIntervalSeconds(int reconciliationIntervalSeconds) {
         this.reconciliationIntervalSeconds = reconciliationIntervalSeconds;
     }
-
+    
     @Description("Timeout for the ZooKeeper session")
     @Minimum(0)
     @JsonInclude(JsonInclude.Include.NON_DEFAULT)
-    public int getZookeeperSessionTimeoutSeconds() {
+    @DeprecatedProperty(description = "This property has been deprecated because ZooKeeper is not used anymore by the Topic Operator.")
```

Comment:
Same as above. You should put more focus on it being ignores and not used anymore rather than it being deprecated.